docker/task.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:22.04 AS base
+
+FROM base AS builder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN ( apt-get update && yes | unminimize ) && \
```

Comment:
Was there something in particular that `unminimize` pulls in?  Otherwise I expect this will end up downloading a bunch of unused stuff?

Reply:
The usage of `uniminimize` stems from the Timewarrior (test) image. Without it, you get the following message when calling e.g. man pages:
```
This system has been minimized by removing packages and content that are not required on a system that users do not log into.
To restore this content, including manpages, you can run the 'unminimize' command. You will still need to ensure the 'man-db' package is installed.
```
So I guess one can remove this from the builder image.

Reply:
Ah, manpages makes sense. And yeah, not necessary for the temporary builder image :)